crates/rspack_plugin_html/src/config.rs

```diff
@@ -9,11 +9,13 @@ use sugar_path::SugarPath;
 use crate::sri::HtmlSriHashFunction;
 
 #[cfg_attr(feature = "testing", derive(JsonSchema))]
-#[derive(Deserialize, Debug, Clone, Copy)]
+#[derive(Deserialize, Debug, Clone, Copy, PartialEq)]
```

Comment:
Why add `PartialEq`?

Reply:
When inject is 'true' or 'false', conditional operators need to be used for judgment processing.

Reply:
where is the related code? Could you point it to me?

Reply:
> where is the related code? Could you point it to me?

ok